src/main/kotlin/dev/avt/api/admin/absence/readRequestedHours/HoursRequestedDataFormat.kt

```diff
@@ -4,4 +4,4 @@ import kotlinx.serialization.Serializable
 
 // TODO add magister grade and test data
 @Serializable
-data class HoursRequestedDataFormat(val user: String, val startTime: Long, val endTime: Long, val id: Int)
\ No newline at end of file
+data class HoursRequestedDataFormat(val user: Int, val startTime: Long, val endTime: Long, val timeRequested: Long, val id: Int)
```

Comment:
wat doet `timeRequested` hier? is dat niet gewoon `endTime-startTime`

Reply:
de tijd dat de user de absentie heeft aangevraagd

Reply:
ah, prima

zou het voortaan `requestTime` noemen